xdis/__init__.py

```diff
@@ -30,6 +30,7 @@
 __docformat__ = 'restructuredtext'
 
 PYTHON3 = (sys.version_info >= (3, 0))
+USE_WORDCODE = (sys.version_info >= (3, 6))
```

Comment:
I wonder if this will ever be used? It might make sense in the test,  but for other things, wordcode is about which python version you are working off of, and not about the interpreter that is running this.